src/ontology/imports/foodon_product_import.owl

```diff
@@ -148852,6 +148870,12 @@ size larger than 100 mesh, U.S. Standard Sieve Size.</obo:IAO_0000115>
 
     <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_03311654">
         <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00001696"/>
+        <rdfs:subClassOf>
```

Comment:
Delete this and instead put derives from bos taurus on the 'beef food product' parent.  All children of 'beef food product' don't need a derives from clause any more then.

Reply:
Okay done